app/models/application_form.rb

```diff
@@ -119,6 +119,10 @@ def any_offers?
     application_choices.map.any?(&:offer?)
   end
 
+  def application_not_sent?
```

Comment:
Perhaps this, to differentiate it from `ApplicationChoice#application_not_sent?`

```suggestion
  def all_applications_not_sent?
```